abe/unittest.py

```diff
@@ -89,6 +89,21 @@ def assert_data_list_equal(self, data1, data2):
             )
             raise type(exceptions[0])(message)
 
+    def assert_headers_equal(self, response_data, spec_data):
```

Comment:
Docstring?

I think this implementation suggests that what we are actually checking is `assert_headers_include`... or are a superset of those in the sample?